flight/Modules/Stabilization/stabilization.c

```diff
@@ -354,7 +354,35 @@ static void stabilizationTask(void* parameters)
 
 					break;
 
-				case STABILIZATIONDESIRED_STABILIZATIONMODE_ATTITUDE:
+			case STABILIZATIONDESIRED_STABILIZATIONMODE_ACROPLUS:
+					// this implementation is based on the Openpilot/Librepilot Acro+ flightmode
+					// and our existing rate & MWRate flightmodes
+					if(reinit)
+							pids[PID_RATE_ROLL + i].iAccumulator = 0;
+
+					// The unscaled input (-1,1)
+					float *raw_input = &stabDesired.Roll;
+
+					// The factor for gyro suppression / mixing raw stick input into the output; scaled by raw stick input
+					float factor = fabsf(raw_input[i]) * settings.AcroInsanityFactor / 100;
+
+					// Store to rate desired variable for storing to UAVO
+					rateDesiredAxis[i] = bound_sym(raw_input[i] * settings.ManualRate[i], settings.ManualRate[i]);
+
+					// Zero integral for aggressive maneuvers, like it is done for MWRate
+					if ((i < 2 && fabsf(gyro_filtered[i]) > 150.0f) ||
+											(i == 0 && fabsf(raw_input[i]) > 0.2f)) {
+							pids[PID_RATE_ROLL + i].iAccumulator = 0;
+							pids[PID_RATE_ROLL + i].i = 0;
+							}
+
+					// Compute the inner loop
+					actuatorDesiredAxis[i] = pid_apply_setpoint(&pids[PID_RATE_ROLL + i], rateDesiredAxis[i], gyro_filtered[i], dT);
+					actuatorDesiredAxis[i] = bound_sym(actuatorDesiredAxis[i], 1.0f);
```

Comment:
move this line down one. you really want to make sure the final computed actuatorDesired is limited to -1,1